server/src/main/java/org/elasticsearch/search/query/QueryPhase.java

```diff
@@ -92,7 +163,7 @@ public static void execute(SearchContext searchContext) throws QueryPhaseExecuti
      * In a package-private method so that it can be tested without having to
      * wire everything (mapperService, etc.)
      */
-    static void executeInternal(SearchContext searchContext) throws QueryPhaseExecutionException {
+    static void executeWithCollectors(SearchContext searchContext) throws QueryPhaseExecutionException {
```

Comment:
I am not sold on the new name here, because the collectors get created later on as part of the searchWithCollector method. I must admit I cannot come up with anything better than executeQueryInternal so far :)

Reply:
I think this is similar to what it was called before? But I admit the name isn't great here. Open to suggestions :)

Reply:
I changed this to `addCollectorsAndExecuteQuery`